hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/solr/SolrUtil.java

```diff
@@ -252,52 +255,62 @@ private void updateBucketSize(EntityReadAccessHeatMapResponse bucket,
     bucket.setSize(bucket.getSize() + keySize);
   }
 
-  public void queryLogs(String path, String entityType, String startDate,
-                        SolrHttpClient solrHttpClient) {
+  public AtomicReference<EntityReadAccessHeatMapResponse>
```

Comment:
Hi @devmadhuu, since we return the result, I think we can just return EntityReadAccessHeatMapResponse instead of an AtomicReference.

Reply:
> Hi @devmadhuu, since we return the result, I think we can just return EntityReadAccessHeatMapResponse instead of an AtomicReference.

Thanks @ChenSammi , Pls have a look again. I have done changes.